.github/workflows/build-system-test.yml

```diff
@@ -55,9 +55,9 @@ jobs:
             language: ts
             node-version: 20
           - framework: react
-            framework-version: 17
+            framework-version: 18
             build-tool: next
-            build-tool-version: 11
+            build-tool-version: 12
```

Comment:
Since Next 14 has dropped, we should focus on testing Next 12, 14 rather than 11. Next.js 11 doesn't work with Node 20.

Reply:
Note, we will still get coverage for React 16 under our CRA tests.